File: Iniciante/1020/1020.c
#include <stdio.h>

int main(){
    int x, anos, meses, dias;
    scanf("%d", &x);
    anos = x/365;
    meses = (x%365)/30;
    dias = (x%365)%30;
    printf("%d ano(s)\n", anos);
    printf("%d mes(es)\n", meses);
    printf("%d dia(s)\n", dias);
    return 0;
}

File: Iniciante/1020/1020.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner i = new Scanner(System.in);
        int dias = i.nextInt();
        int ano = dias/365;
        int mes = (dias%365)/30;
        int dia = (dias%365)%30;
        System.out.printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n", ano, mes, dia);
    }
}